### PR TITLE
Bug 1647168: Peer review comments entered

### PR DESCRIPTION
**Pull request by jhoyt-rh**
This PR is for BZ1647168 (https://bugzilla.redhat.com/show_bug.cgi?id=1647168)

This PR is a new PR to fix some issues with the previous PR # 12894. Please see that PR (https://github.com/openshift/openshift-docs/pull/12894), which I will close as a duplicate. 

All Peer review comments are shown in PR # 12894. Those comments were addressed previously and are included in this PR. For this reason, this PR has the label Peer Review Done. What remains is for QE to review this PR. 

@ jianlinlui PTAL
@ gkimetto PTAL

### CODE REVIEW

**Comment by jianlinliu** [comment]
@anpingli Pls help review the PR.

**Comment by anpingli** [comment]
LGTM

**Comment by kimettog** [comment]
Looks good

File: install_config/aggregate_logging.adoc [hunk 1]
```diff
@@ -851,9 +849,22 @@ spec:
...
+For example, if your deployment has three infrastructure nodes, you could add labels for those
+nodes as follows:
+----
+$ oc label node <nodename1> logging-es-node=1
+$ oc label node <nodename2> logging-es-node=2
+$ oc label node <nodename3> logging-es-node=3
+----
+
+For information about adding a label to a node, see
+xref:../admin_guide/manage_nodes.html#updating-labels-on-nodes[Updating Labels on Nodes].
```
Comment (vikram-redhat) [inline install_config/aggregate_logging.adoc]: Hey @jhoyt-rh xrefs should link to adoc files, not html.

**Pull request merged**